src/QSymbolicsBase/QSymbolicsBase.jl

```diff
@@ -171,10 +173,10 @@ propsequal(x,y) = all(n->isequal(getproperty(x,n),getproperty(y,n)), propertynam
 ##
 
 include("literal_objects.jl")
+include("predefined_CPTP.jl")
 include("basic_ops_homogeneous.jl")
 include("basic_ops_inhomogeneous.jl")
 include("predefined.jl")
-include("predefined_CPTP.jl")
```

Comment:
In my mind "predefined" is meant for some well known specific instances of symbolic types, like the number pi, or the Pauli operator X, not for types that can encompass many instances. Maybe it would make sense to move some of the stuff out from this file and into a new `basic_superops.jl`?

Reply:
I was thinking about this too and agree. I think we should also move linear algebra operations like `dagger` and `trace` out of `predefined.jl` into a new file, perhaps 'linalg.jl`?

Reply:
good idea. I like the shorter name.